Include/pythonrun.h

```diff
@@ -32,60 +32,9 @@ PyAPI_FUNC(int) PyRun_InteractiveLoopFlags(
     const char *filename,       /* decoded from the filesystem encoding */
     PyCompilerFlags *flags);
 
-PyAPI_FUNC(struct _mod *) PyParser_ASTFromString(
```

Comment:
We cannot remove these functions as they are part of the public API. We need to provide new implementations

Reply:
What if we make macros out of them and just call the corresponding `PyPegen_AST*` function? Would that work?

Reply:
We could do this in a separate PR if you wish

Reply:
> What if we make macros out of them and just call the corresponding PyPegen_AST* function? Would that work?

We could rename the PyPegen_AST functions better I think

Reply:
> We could rename the PyPegen_AST functions better I think

Won't there be folks that use the `PyPegen_*` functions in 3.9, whose code will break as well in that case?

Reply:
I propose to remove these changes in this PR and dealing with the implementations in other

Reply:
> Won't there be folks that use the `PyPegen_*` functions in 3.9, whose code will break as well in that case?

Are those part of the Public API? If so, we should make them private I think

Reply:
> I propose to remove these changes in this PR and dealing with the implementations in other

Yup, done in https://github.com/python/cpython/pull/21005/commits/1a86551632669c4062513fae1b2f26912fd83ca0.